README.md

```diff
@@ -43,37 +43,22 @@ Istio configures an envoy proxy using a set of
 We can hook into this mechanism and insert additional configuration, which
 further limits the access to a specific cluster.
 
-Broadly speaking, there are three different external traffic flows:
+Broadly speaking, there are two different external traffic flows:
```

Comment:
Can you reference GEP-30 for more details, especially on how this changed recently?